options/locale/locale_en-US.ini

```diff
@@ -1296,6 +1296,12 @@ pulls.update_not_allowed = You are not allowed to update branch
 pulls.outdated_with_base_branch = This branch is out-of-date with the base branch
 pulls.closed_at = `closed this pull request <a id="%[1]s" href="#%[1]s">%[2]s</a>`
 pulls.reopened_at = `reopened this pull request <a id="%[1]s" href="#%[1]s">%[2]s</a>`
+pulls.merge_instruction_hint = You can also view
+pulls.merge_instruction_link_text = command line instructions
```

Comment:
Hmm, can we combine this into one entry? If split apart, it might result in wrong sounding sentences. Also, finish sentence with `.`.